src/main/java/com/google/cloud/tools/managedcloudsdk/ManagedCloudSdk.java

```diff
@@ -167,7 +168,14 @@ public boolean isUpToDate() throws ManagedSdkVerificationException {
   // TODO : fix passthrough for useragent and client side usage reporting
   public SdkInstaller newInstaller() {
     String userAgentString = "google-cloud-tools-java";
-    return SdkInstaller.newInstaller(managedSdkDirectory, version, osInfo, userAgentString, false);
+    return SdkInstaller.newInstaller(
+        managedSdkDirectory, version, osInfo, userAgentString, false, Collections.emptyMap());
+  }
+
+  public SdkInstaller newInstaller(Map<String, String> environmentVariables) {
+    String userAgentString = "google-cloud-tools-java";
```

Comment:
Is this ever set to anything else? If not, it could be a constant.

Reply:
Makes sense, will do.